### PR TITLE
Use Dependabot for the PHP images

### PR DESCRIPTION
**Pull request by arnested**
According to https://github.com/dependabot/dependabot-core/issues/1971
and https://github.com/dependabot/dependabot-core/pull/6628 it should
be possible to update only the SHA's now.

We'll leave our existing update mechanism in place until we have seen
whether Dependabot will actually do what we expect / hope for.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -10,7 +10,9 @@ updates:
       - "reload/developers"
     ignore:
       - dependency-name: "php"
+        update-times: ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
```
Comment (deivid-rodriguez) [inline .github/dependabot.yml:13]: You got a typo in there, it's `update-types`, not `update-times`!

Comment (arnested) [reply .github/dependabot.yml:13]: Yeah, I'm apparently very bad at copy'n'paste. I also discovered it and fixed it in a86287ca260a6cc64a5be11d40c16e6b28a46929.

Comment (deivid-rodriguez) [reply .github/dependabot.yml:13]: Oh right, sorry I didn't notice you had already fixed it! And I see now that Dependabot created a (hopefully correct?) PR!

Comment (arnested) [reply .github/dependabot.yml:13]: Yeah. I think it's correct (haven't checked in details yet). 

I created a copy earlier with digests that definitely needed updating.

Dependabot created a correct pull request for the first occurrence of the php image (`php:8.0-fpm-alpine`). 

But it didn't create pull requests for the next two occurrences of the same image in different versions (`php:8.1-fpm-alpine` and `php:8.2-fpm-alpine`): https://github.com/arnested/dep-php-test/pull/1

That's probably not a problem with the new change, but an edge case that will only show it's face now).
